pyrefly/lib/error/kind.rs

```diff
@@ -77,6 +77,8 @@ pub enum ErrorKind {
     BadUnpacking,
     /// Attempting to `del` something that cannot be deleted
     DeleteError,
+    /// An error rasied if a function that is deprecated is called.
```

Comment:
"Calling a function marked with @deprecated"

is probably more concise. We can edit the comment later once we add the import check